superagi/controllers/models_controller.py

```diff
@@ -0,0 +1,85 @@
+from fastapi import APIRouter, Depends, HTTPException
+from superagi.helper.auth import check_auth, get_user_organisation
+from superagi.helper.models_helper import ModelsHelper
+from fastapi_jwt_auth import AuthJWT
+from fastapi_sqlalchemy import db
+import logging
+from pydantic import BaseModel
+
+router = APIRouter()
+
+class ValidateAPIKeyRequest(BaseModel):
+    model_provider: str
+    model_api_key: str
+
+class StoreModelRequest(BaseModel):
+    model_name: str
+    description: str
+    end_point: str
+    model_provider_id: int
+    token_limit: int
+    type: str
+
+@router.post("/storeApiKeys", status_code=200)
+async def storeApiKeys(request: ValidateAPIKeyRequest, organisation=Depends(get_user_organisation)):
+    try:
+        return ModelsHelper(session=db.session, organisation_id=organisation.id).storeApiKey(request.model_provider, request.model_api_key)
+    except Exception as e:
+        logging.error(f"Error while storing API key: {str(e)}")
+        raise HTTPException(status_code=500, detail="Internal Server Error")
+
+@router.get("/getApiKeys")
```

Comment:
api endpoints not standard. 

GET <>/api_keys
POST <>/api_key
GET <>/api_key
POST /verify_end_point
POST <>/model
GET <>/models
GET <>/model/:model_id
GET <>/model_data/:model

Reply:
done